### PR TITLE
build: enable OS X RBE via rules_nodejs update and local browser_repositories management

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

**Comment by josephperrott** [comment]
Caretaker Note: It might be best for all of these commits to go in individually.

File: scripts/local-dev/setup-rbe.sh [hunk 1]
```diff
@@ -1,71 +1,100 @@
...
+###########################################################
+# The full path location of the script
+full_script_path="$(pwd)/$(dirname ${BASH_SOURCE[0]})"
 # Determine the root directory of the Angular github repo.
-github_directory_root=$(git rev-parse --show-toplevel 2> /dev/null);
+project_directory=$(git rev-parse --show-toplevel 2> /dev/null)
 if [[ $? -ne 0 ]]; then
-  echo "This command must be run from within the cloned \"angular/components\" repository."
-  exit 1;
+  echo "This command must be run from within the cloned \"angular/angular\" repository"
```
Comment (devversion) [inline scripts/local-dev/setup-rbe.sh]:
```suggestion
  echo "This command must be run from within the cloned \"angular/components\" repository"
```

File: tools/browsers/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
```
Comment (devversion) [inline tools/browsers/BUILD.bazel]: Can we have a readme that briefly captures what you wrote in `browser_repositories.bzl`? (can be just copied)

Also, are there plans to move this change upstream? link to an issue if present?

File: scripts/local-dev/setup-rbe.sh [hunk 3]
```diff
@@ -1,71 +1,100 @@
 #!/bin/bash
 # A script for automatically configuring a user's local dev
 # environment to use Remote Build Execution.
+# Short cuts to set output as bold and normal
```
Comment (jelbourn) [inline scripts/local-dev/setup-rbe.sh]: Add blank line

File: tools/browsers/BUILD.bazel [hunk 4]
```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 Google Inc.
```
Comment (jelbourn) [inline tools/browsers/BUILD.bazel]: Seems like the wrong license?

**Pull request closed without merging**